extensions/guacamole-auth-duo/src/main/resources/translations/it.json

```diff
@@ -0,0 +1,13 @@
+{
+
+    "DATA_SOURCE_DUO" : {
+        "NAME" : "Duo TFA Backend"
+    },
+
+    "LOGIN" : {
+        "FIELD_HEADER_GUAC_DUO_SIGNED_RESPONSE" : "",
```

Comment:
You can leave out empty translation strings - there's no reason to include them.

Reply:
```
{

    "DATA_SOURCE_DUO" : {
        "NAME" : "Duo TFA Backend"
    },

    "LOGIN" : {
        "FIELD_HEADER_GUAC_DUO_SIGNED_RESPONSE" : "",
        "INFO_DUO_VALIDATION_CODE_INCORRECT"    : "Codice di convalida Duo errato.",
        "INFO_DUO_AUTH_REQUIRED"                : "Si prega di autenticarsi con Duo per continuare."
    }

}
```

Hello @necouchman this is the content of mine it.json in my forked repo (the one I've pulled) so I don't know why you see an empty translation string; what can I do to fix this?